### PR TITLE
Add semicolons

### PR DESCRIPTION
**Pull request by YorkshireKev**
Just added semicolons to statements in some of the examples to make the documentation more consistent as some code had them while others didn't.

### CODE REVIEW

File: guide/migrating-4.md [hunk 1]
```diff
@@ -134,13 +134,13 @@ Here is an example of chained route handlers defined using `app.route()`.
 app.route('/book')
   .get(function(req, res) {
     res.send('Get a random book');
-  })
+  });
```
Comment (dougwilson) [inline guide/migrating-4.md:137]: this is wrong; it's a syntax error

File: guide/migrating-4.md [hunk 2]
```diff
@@ -134,13 +134,13 @@ Here is an example of chained route handlers defined using `app.route()`.
...
   .post(function(req, res) {
     res.send('Add a book');
-  })
+  });
```
Comment (dougwilson) [inline guide/migrating-4.md:140]: this is wrong; it's a syntax error

File: guide/migrating-4.md [hunk 3]
```diff
@@ -134,13 +134,13 @@ Here is an example of chained route handlers defined using `app.route()`.
...
   .put(function(req, res) {
     res.send('Update the book');
-  })
+  });
```
Comment (dougwilson) [inline guide/migrating-4.md]: this is wrong; it's a syntax error

File: guide/routing.md [hunk 4]
```diff
@@ -228,13 +228,13 @@ Here is an example of chained route handlers defined using `app.route()`.
 app.route('/book')
   .get(function(req, res) {
     res.send('Get a random book');
-  })
+  });
   .post(function(req, res) {
     res.send('Add a book');
-  })
+  });
```
Comment (dougwilson) [inline guide/routing.md:234]: this is wrong; it's a syntax error

File: guide/routing.md [hunk 5]
```diff
@@ -228,13 +228,13 @@ Here is an example of chained route handlers defined using `app.route()`.
 app.route('/book')
   .get(function(req, res) {
     res.send('Get a random book');
-  })
+  });
```
Comment (dougwilson) [inline guide/routing.md:231]: this is wrong; it's a syntax error

File: guide/migrating-4.md (hunk 1 above)
Comment (YorkshireKev) [inline guide/migrating-4.md:137]: @dougwilson you're right. Seems that I've been over-keen on adding the semicolons. I'll fix and make another pull request.

**Pull request merged**